package.json

```diff
@@ -7,7 +7,7 @@
     "test": "npm run lint && npm run unit && npm run cover && npm run snyk",
     "lint": "eslint .",
     "unit": "mocha test/ --require ./test/helpers --recursive",
-    "cover": "istanbul cover _mocha -- test/ --recursive --require test/helpers",
+    "cover": "istanbul cover _mocha -- -R dot test/ --recursive --require test/helpers",
```

Comment:
makes istanbul test less verbose by showing dots instead of the full test results